pkg/apis/build/v1alpha1/clusterbuildstrategy_types.go

```diff
@@ -85,6 +85,11 @@ func (s ClusterBuildStrategy) GetVolumes() []BuildStrategyVolume {
 	return s.Spec.Volumes
 }
 
+// GetLabels returns the labels of the build strategy
+func (s ClusterBuildStrategy) GetLabels() map[string]string {
+	return s.Labels
+}
```

Comment:
I don't think this is needed because Kubernetes already provides that function on `ObjectMeta`.